ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.12.7
+    tag: 1.13.9
 
 inputs:
   - name: dp-deployer
```

Comment:
After this line you need to remove the path: as it should no longer be located in the GOPATH with modules.
Also you can add in a cache directive so concourse doesn't download the dependencies on every build.
Rest of the file should look like this…
```
inputs:
  - name: dp-deployer

outputs:
  - name: build

caches:
  - path: go/

run:
  path: dp-deployer/ci/scripts/build.sh